hw/ip/hmac/dv/cov/hmac_cov_unr_excl.el

```diff
@@ -0,0 +1,141 @@
+Condition 16 "4228513916" "((((reqfifo_rdata.op == OpRead) & (~reqfifo_rdata.error))) ? reqfifo_rready : 1'b0) 1 -1" (2 "1")
+CHECKSUM: "1413060523 1581044118"
+INSTANCE: tb.dut.u_sha2
+Fsm sha_st_q "2028159779"
+ANNOTATION: "VC_COV_UNR"
+Transition ShaUpdateDigest->ShaCompress "2->1"
+CHECKSUM: "2024123546 2243903072"
+INSTANCE: tb.dut.u_tlul_adapter.u_reqfifo
+ANNOTATION: "VC_COV_UNR"
+Branch 0 "1862733684" "gen_normal_fifo.full" (2) "gen_normal_fifo.full 0,0"
+ANNOTATION: "VC_COV_UNR"
+Branch 3 "2807735941" "(!rst_ni)" (1) "(!rst_ni) 0,1,-,-"
+ANNOTATION: "VC_COV_UNR"
+Branch 3 "2807735941" "(!rst_ni)" (3) "(!rst_ni) 0,0,1,0"
+ANNOTATION: "VC_COV_UNR"
+Branch 4 "1287846560" "(!rst_ni)" (1) "(!rst_ni) 0,1,-,-"
+ANNOTATION: "VC_COV_UNR"
+Branch 4 "1287846560" "(!rst_ni)" (3) "(!rst_ni) 0,0,1,0"
+CHECKSUM: "2024123546 2243903072"
+INSTANCE: tb.dut.u_tlul_adapter.u_sramreqfifo
+ANNOTATION: "VC_COV_UNR"
+Branch 0 "1862733684" "gen_normal_fifo.full" (0) "gen_normal_fifo.full 1,-"
+ANNOTATION: "VC_COV_UNR"
+Branch 0 "1862733684" "gen_normal_fifo.full" (2) "gen_normal_fifo.full 0,0"
+ANNOTATION: "VC_COV_UNR"
+Branch 1 "3867317506" "gen_normal_fifo.empty" (1) "gen_normal_fifo.empty 0"
+ANNOTATION: "VC_COV_UNR"
+Branch 3 "2807735941" "(!rst_ni)" (1) "(!rst_ni) 0,1,-,-"
+ANNOTATION: "VC_COV_UNR"
+Branch 3 "2807735941" "(!rst_ni)" (2) "(!rst_ni) 0,0,1,1"
+ANNOTATION: "VC_COV_UNR"
+Branch 3 "2807735941" "(!rst_ni)" (3) "(!rst_ni) 0,0,1,0"
+ANNOTATION: "VC_COV_UNR"
+Branch 4 "1287846560" "(!rst_ni)" (1) "(!rst_ni) 0,1,-,-"
+ANNOTATION: "VC_COV_UNR"
+Branch 4 "1287846560" "(!rst_ni)" (2) "(!rst_ni) 0,0,1,1"
+ANNOTATION: "VC_COV_UNR"
+Branch 4 "1287846560" "(!rst_ni)" (3) "(!rst_ni) 0,0,1,0"
+ANNOTATION: "VC_COV_UNR"
+Branch 5 "2429929352" "gen_normal_fifo.fifo_incr_wptr" (0) "gen_normal_fifo.fifo_incr_wptr 1"
+CHECKSUM: "2024123546 4157704780"
+INSTANCE: tb.dut.u_tlul_adapter.u_rspfifo
+ANNOTATION: "VC_COV_UNR"
+Branch 0 "1862733684" "gen_normal_fifo.full" (0) "gen_normal_fifo.full 1,-"
+ANNOTATION: "VC_COV_UNR"
+Branch 0 "1862733684" "gen_normal_fifo.full" (2) "gen_normal_fifo.full 0,0"
+ANNOTATION: "VC_COV_UNR"
+Branch 1 "4156807255" "(gen_normal_fifo.fifo_empty && wvalid_i)" (0) "(gen_normal_fifo.fifo_empty && wvalid_i) 1"
+ANNOTATION: "VC_COV_UNR"
+Branch 2 "3867317506" "gen_normal_fifo.empty" (1) "gen_normal_fifo.empty 0"
+ANNOTATION: "VC_COV_UNR"
+Branch 4 "2807735941" "(!rst_ni)" (1) "(!rst_ni) 0,1,-,-"
+ANNOTATION: "VC_COV_UNR"
+Branch 4 "2807735941" "(!rst_ni)" (2) "(!rst_ni) 0,0,1,1"
+ANNOTATION: "VC_COV_UNR"
+Branch 4 "2807735941" "(!rst_ni)" (3) "(!rst_ni) 0,0,1,0"
+ANNOTATION: "VC_COV_UNR"
+Branch 5 "1287846560" "(!rst_ni)" (1) "(!rst_ni) 0,1,-,-"
+ANNOTATION: "VC_COV_UNR"
+Branch 5 "1287846560" "(!rst_ni)" (2) "(!rst_ni) 0,0,1,1"
+ANNOTATION: "VC_COV_UNR"
+Branch 5 "1287846560" "(!rst_ni)" (3) "(!rst_ni) 0,0,1,0"
+ANNOTATION: "VC_COV_UNR"
+Branch 6 "2429929352" "gen_normal_fifo.fifo_incr_wptr" (0) "gen_normal_fifo.fifo_incr_wptr 1"
```

Comment:
Because the rspfifo is always empty, so wptr never increment. 
Excluded code: https://github.com/lowRISC/opentitan/blob/master/hw/ip/prim/rtl/prim_fifo_sync.sv#L137